app/Gemfile

```
@@ -7,6 +7,15 @@ ruby ">= 3.2"
 gem "cocoapods", ">= 1.13", "!= 1.15.0", "!= 1.15.1"
 gem "activesupport", ">= 6.1.7.5", "!= 7.1.0"
 
+gem 'xcodeproj', '< 1.26.0'
+gem 'concurrent-ruby', '< 1.3.4'
+
+# Ruby 3.4.0 has removed some libraries from the standard library.
+gem 'bigdecimal'
+gem 'logger'
+gem 'benchmark'
```

RuboCop (1.75.5)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `benchmark` should appear before `logger`.

(Bundler/OrderedGems)